### ISSUE TITLE
Array and string offset access syntax with curly braces is no longer supported

### ISSUE THREAD

**Issue by davidpsc** [phpmyadmin/phpmyadmin#17721]
I've updated PHP and phpmyadmin and at first it seems that everything works fine, but when trying to insert a new row into a table (via "Insert" menu, fill out fields and click "Ok") I get a blank screen (no message or content, just phpmyadmin header, navigation bar and footer) and the nginx error log says:
`PHP message: PHP Fatal error:  Array and string offset access syntax with curly braces is no longer supported in /var/www/html/david/phpmyadmin/frontend/libraries/Util.class.php on line 2095`

Insert via SQL statement works fine.

Webserver: nginx/1.18.0
Database client: libmysql - mysqlnd 8.1.2
PHP version: 8.1.2
phpmyadmin version: 5.2.0
Server (OS): 8.0.30-0ubuntu0.22.04.1

**Comment by williamdes** [comment]
Hi @davidpsc

The error you are reporting is probably already solved, could you try the latest 5.2 version in development [(phpMyAdmin 5.2+snapshot)](https://www.phpmyadmin.net/downloads/#snapshot_5.2+snapshot)?
About the error log you sent it seems wrong because:
 - 5.2.0 has so such file
 - only old versions of phpMyAdmin had files named `.class.php`
 - I can not find any similar reports about such a file

This leads me to think that maybe you where using a very old version or it's a class from your website. 
Anyway let's see what the snapshot version does on your end ;)

**Comment by davidpsc** [comment]
Hi @williamdes

Thanks for your reply!

I can confirm it is working with the 5.2 snapshot version (I tested 5.2.1-dev+20220911.94f0d931cd). I double-checked with 5.2.0 but the error is reproducible in that version.

I think the log entry was from the old phpmyadmin version before I upgraded it (I tried calling my phpmyadmin 4.x installation after upgrading PHP to 8.1) so never mind. Now I don't see any errors being logged although the blank page still occurs with 5.2.0 as described. It seems that the "Insert" submission only fails for a single table. For other tables it works fine even with 5.2.0.

If there is no negative impact of using the snapshot version I will stick with 5.2.1-dev... for now as it seems everything is working fine with this version.

Thank you very much!
